.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ executors:
   reactnativeios:
     <<: *defaults
     macos:
-      xcode: &_XCODE_VERSION "12.5.0"
+      xcode: &_XCODE_VERSION "13.0.0"
```

Comment:
actually we should be really using 13.3.1 at this point

Reply:
This doesn't matter anyway since we use ADO for our CI checks, but sure, it doesn't hurt to change this as well.